Allclean

```diff
@@ -4,15 +4,15 @@
 echo "Cleaning logs and output files from previous runs..."
 
 rm -f precice-FLUID-events.json \
-      precice-STRUCTURE-events.json \
+      precice-Solid-events.json \
       precice-FLUID-iterations.log \
-      precice-STRUCTURE-convergence.log \
-      precice-STRUCTURE-iterations.log \
+      precice-Solid-convergence.log \
+      precice-Solid-iterations.log \
       precice-FLUID-events-summary.log \
       precice-STRUCTUR-events-summary.log \
       Postproc/*.vtk \
       Fluid.log \
-      Structure.log
+      Solid.log
 
 rm -r precice-run/
 
```

Comment:
Missing one file:

```suggestion
rm -rfv precice-Solid-events-summary.log
```

The `-v` (verbose) option also displays when a file is deleted, the `-f` (force) does not complain if the file is not there.

Reply:
sorry, the -rfv flag should be used for the other files or only this missing one?

Reply:
It should be also for the rest. But don't worry too much about the `Allclean`, I will anyway try to make a general one.

Reply:
okay thanks!